polylogue/config.py

```
@@ -70,6 +76,9 @@ def import_claude(self) -> Path:
 @dataclass
 class Config:
     defaults: Defaults = field(default_factory=Defaults)
+    drive: Optional[DriveConfig] = None
```

Ruff (0.15.9)

[error] Undefined name `DriveConfig`

(F821)